tests/test_base.py

```diff
@@ -35,6 +35,20 @@ def test_passwd_present(auto_container):
     assert auto_container.connection.file("/etc/passwd").exists
 
 
+@pytest.mark.skipif(
+    OS_VERSION not in ("tumbleweed",),
+    reason="requires glibc-locale-base installed",
+)
+def test_iconv_working(auto_container):
+    """Generic test iconv works for UTF8 and ISO-8859-15 locale"""
+    assert (
+        auto_container.connection.check_output(
+            "echo -n 'SüSE' | iconv -f UTF8 -t ISO_8859-15 | wc -c"
```

Comment:
Won't the pipe swallow the return code of iconv?

Reply:
it does, but the wc -c is unlikely to be 4 in case of failure. we can also remove the wc -c to capture the exit code, not sure if that is better.

Reply:
> but the wc -c is unlikely to be 4 in case of failure

Indeed, it'll probably be fine